### DISCUSSION TITLE
inconsistency in interface style or display problems when switching to the console?

### DISCUSSION THREAD

**Discussion by codiggermkt** [zed-industries/zed#9472] [Feature Requests]
Will the installation fail to work properly on certain operating system versions? Are there any requirements for a specific operating system version or dependent library? Are there compatibility issues during installation?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
